sympy/functions/combinatorial/numbers.py

```diff
@@ -1984,3 +1984,102 @@ def nT(n, k=None):
     for discard in m.enum_range(n[_M], k-1, k):
         tot += 1
     return tot
+
+#-----------------------------------------------------------------------------#
+#                                                                             #
+#                          Motzkin numbers                                    #
+#                                                                             #
+#-----------------------------------------------------------------------------#
+class motzkin(Function):
+    """
+    In mathematics, the nth Motzkin number is the number
+    of different ways of drawing non-intersecting chords
+    between n points on a circle (not necessarily touching
+    every point by a chord). The Motzkin numbers are named
+    after Theodore Motzkin and have diverse applications
+    in geometry, combinatorics and number theory.
+
+    Motzkin numbers satisfy this recurrence relation
+    M(n) = ( (2n+1) / (n+2) )*M(n-1)  +  ( (3n-3) / (n+2) )*M(n-2)
+    Examples
```

Comment:
Add empty line before Examples

Reply:
Ok , i will do it .

Reply:
You could pretty print this like the ode docstrings do:
```
>>> pprint((M(n - 1)*(2*n + 1) + 3*M(n - 2)*(n - 1))/(n + 2))
3*(n - 1)*M(n - 2) + (2*n + 1)*M(n - 1)
---------------------------------------
                 n + 2  
```